library/js/jquery.treeview-1.4.1/demo/index.html

```diff
@@ -253,8 +253,8 @@ <h4>Sample 3 - fast animations, all branches collapsed at first, red theme, cook
 	
 	<h4>Sample 4 - two trees with one tree control, black and gray theme, cookie-based persistance</h4>
 	<div id="treecontrol">
-		<a title="Collapse the entire tree below" href="#"><img src="../images/minus.gif" /> Collapse All</a>
-		<a title="Expand the entire tree below" href="#"><img src="../images/plus.gif" /> Expand All</a>
+		<a title="Collapse the entire tree below" href="#"><img src="../../../../assets/images/minus.gif" />Collapse All</a>
```

Comment:
You sure this is correct?  I can check.  Mind you, this DEMO file is junk.  Just making sure this is not a blind find and replace.  There are a LOT of features I need to look at to test this PR.